### PR TITLE
Feature/new-account-menu

### PR DESCRIPTION
**Pull request by szymonlesisz**
added framer-motion

### CODE REVIEW

File: packages/suite/src/components/wallet/Menu/components/AccountNavigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    justify-content: center;
+    width: 16px;
+`;
+
+const Text = styled.div`
+    padding-left: 8px;
+`;
+
+interface Props {
+    account: any;
```
Comment (szymonlesisz) [inline packages/suite/src/components/wallet/Menu/components/AccountNavigation/index.tsx]: ?

File: packages/suite/src/components/suite/MenuSecondary/index.tsx [hunk 2]
```diff
@@ -1,88 +1,30 @@
...
-        border-radius: 10px;
-        border: 2px solid #fff;
-    }
-
-    /* set button(top and bottom of the scrollbar) */
-    ::-webkit-scrollbar-button {
-        display: none;
-    }
+    padding-top: 10px;
+    margin: 10px 10px 20px 10px;
```
Comment (mroz22) [inline packages/suite/src/components/suite/MenuSecondary/index.tsx:13]: nitpicks, just that values % 4 !== 0

File: packages/suite/src/components/wallet/Menu/components/AddAccount/index.tsx [hunk 3]
```diff
@@ -1,54 +1,55 @@
...
+    padding: 10px;
+    position: sticky;
+    z-index: 2;
+    bottom: 0;
+    background: ${colors.WHITE};
+    box-shadow: 0 0 14px 0 rgba(0, 0, 0, 0.2);
+    > div:first-of-type {
+        display: block;
+        flex: 1;
+        background: red;
```
Comment (mroz22) [inline packages/suite/src/components/wallet/Menu/components/AddAccount/index.tsx]: isnt that some debugging red?

**Review by mroz22 — APPROVED** [review]

File: packages/suite/src/components/suite/MenuSecondary/index.tsx (hunk 2 above)
Comment (szymonlesisz) [reply packages/suite/src/components/suite/MenuSecondary/index.tsx:13]: i've tried :) 8 is to small 12 is too big :/

**Pull request merged**